src/utils/wallet.js

```diff
@@ -42,6 +57,9 @@ export function WalletProvider({ children }) {
         message: 'Wallet update',
         description: 'Connected to wallet ' + keyToDisplay,
       });
+      wallet
```

Comment:
I'm not sure if we need to handle cluster here, it seems that serum-dex only use the wallet adapter for message/tx signing, not sending. So it does not matter which cluster ezWallet is connecting to.

Reply:
Use cluster information to access explorer.solana.com in testnet/devnet/... to get account details.